src/aind_slims_api/core.py

```diff
@@ -212,35 +132,85 @@ def fetch_models(
         start: Optional[int] = None,
         end: Optional[int] = None,
         **kwargs,
-    ) -> tuple[list[SlimsBaseModelTypeVar], list[dict[str, Any]]]:
+    ) -> list[SlimsBaseModelTypeVar]:
         """Fetch records from SLIMS and return them as SlimsBaseModel objects
 
         Returns
         -------
         tuple:
             list:
                 Validated SlimsBaseModel objects
-            list:
-                Dictionaries representations of objects that failed validation
+
+        Notes
+        -----
+        - kwargs are mapped to field alias values
         """
+        resolved_kwargs = deepcopy(model._base_fetch_filters)
```

Comment:
A warning about accessing a private method. We should probably make it public or expose a public method to fetch this attribute.

Reply:
it's not really information that i want to expose to the end user and is mostly consumed by the client, how do you think we should represent this information on the model or handle this relationship with the client?